src/components/Editor/DebugLine.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+import { Component } from "react";
+import type { SourceEditor } from "devtools-source-editor";
+import type { Frame } from "../../types";
+
+type DebugLineProps = {
+  editor: SourceEditor,
+  frame: Frame,
+  visibleSourceId: string
```

Comment:
Unfortunately, I think we really need data about what source is shown in the editor. It's possible that the source text has loaded but we're at some state where the editor hasn't yet been updated to the new document. (Because that happens in lifecycle callbacks which may _happen_ to occur before or after this component renders with the new source data.)